pkg/rtc/transport.go

```diff
@@ -1478,11 +1478,16 @@ func (t *PCTransport) handleLocalICECandidate(e *event) error {
 	c := e.data.(*webrtc.ICECandidate)
 
 	filtered := false
-	if t.preferTCP.Load() && c != nil && c.Protocol != webrtc.ICEProtocolTCP {
-		cstr := c.String()
-		t.params.Logger.Debugw("filtering out local candidate", "candidate", cstr)
-		t.filteredLocalCandidates = append(t.filteredLocalCandidates, cstr)
-		filtered = true
+	if c != nil {
+		if t.preferTCP.Load() && c.Protocol != webrtc.ICEProtocolTCP {
+			cstr := c.String()
+			t.params.Logger.Debugw("filtering out local candidate", "candidate", cstr)
+			t.filteredLocalCandidates = append(t.filteredLocalCandidates, cstr)
```

Comment:
Do you want add the filtered candidate below to this list so that we have logs? Same in the remote candidate path below also.

Reply:
I was thinking not.. because ideally we would disable them entirely within Pion. logging would be helpful if they had value.. but I'm not seeing any. wdyt?

Reply:
Sounds good. I always err on the side of having more information. That is because I just don't trust myself 🙁 . Jokes apart, it is also due to not knowing all possible values. I have had instances where I look at the log and go "hmm, that should not be there, I need to add this extra `if` to filter that too". So, I err on the side of having the information at the expense of having to read a bit more and a slightly increased cognitive load when reading those logs.

Reply:
I agree with that in general. always having more information.. though going to skip the log here because we are essentially allowing this to return to "default behavior". I'll post a note in the Pion list to let folks know we need to at least offer a setting to disable.